lib/spack/spack/test/container/test_cli.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
```

Comment:
This file is likely to be moved under `test/cmd` but before doing that I just wanted to show that by keeping it here we can move to a more localized scope the fixtures that are only needed to test container image generation (and avoid to put every fixture in `test/conftest.py`). Maybe it's something we could consider in the future?